modules/conversion-center/base/conversion-center-controls-trait.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+namespace Elementor\Modules\ConversionCenter\Base;
+
+use Elementor\Controls_Manager;
+
+trait Conversion_Center_Controls_Trait {
+
+	protected $border_width_range = [
+		'min' => 0,
+		'max' => 10,
+		'step' => 1,
+	];
+
+	protected function add_html_tag_control( string $name ): void {
+		$this->add_control(
+			$name,
+			[
+				'label' => esc_html__( 'HTML Tag', 'elementor' ),
+				'type' => Controls_Manager::SELECT,
+				'options' => [
+					'h1' => 'H1',
+					'h2' => 'H2',
+					'h3' => 'H3',
+					'h4' => 'H4',
+					'h5' => 'H5',
+					'h6' => 'H6',
+					'div' => 'div',
+					'span' => 'span',
+					'p' => 'p',
+				],
+				'default' => 'h2',
+			]
+		);
+	}
+
+	protected function add_borders_control(
+		string $prefix,
+		array $show_border_args = [],
+		array $border_width_args = [],
+		array $border_color_args = [],
```

Comment:
```suggestion
		array $border_color_args = []
```
I think this is what's failing the PHPUnit